samples/fabric/fabric-cli/src/commands/interop-helper.ts

```diff
@@ -151,8 +151,9 @@ const command: GluegunCommand = {
           interopFlowResponse.views[0].toObject()
         )}. Interop Flow result: ${interopFlowResponse.result || 'successful'}`
       )
+      const remoteValue = InteroperableHelper.getResponseDataFromView(interopFlowResponse.views[0])
       spinner.succeed(
-        `Called Function ${applicationFunction}. With Args: ${invokeObject.ccArgs} `
+        `Called Function ${applicationFunction}. With Args: ${invokeObject.ccArgs} ${remoteValue}`
```

Comment:
Add comma before ${remoteValue}

Reply:
Comma is already there when printing ${invokeObject.ccArgs}, as thats a list.